lib/jsdom/living/nodes/HTMLTableCellElement-impl.js

```diff
@@ -2,54 +2,74 @@
 
 const HTMLElementImpl = require("./HTMLElement-impl").implementation;
 
-const closest = require("../helpers/traversal").closest;
-const createDOMTokenList = require("../dom-token-list").create;
-const resetDOMTokenList = require("../dom-token-list").reset;
-
-class HTMLTableCellImpl extends HTMLElementImpl {
-  get headers() {
-    if (this._headers === undefined) {
-      this._headers = createDOMTokenList(this, "headers");
-    }
-    return this._headers;
-  }
+const { asciiLowercase } = require("../helpers/strings");
+const { closest } = require("../helpers/traversal");
 
-  get cellIndex() {
-    const tr = closest(this, "tr");
-    if (tr === null) {
-      return -1;
-    }
-
-    return tr.cells.indexOf(this);
+function reflectedAttributeClampedToRange(attrValue, min, max, defaultValue = 0) {
```

Comment:
This looks like it could be useful in other locations. Move to the helpers directory, perhaps? Though eventually we should add `[Reflect]` varieties to webidl2js, I suppose.